modules/project-factory/schemas/project.schema.json

```diff
@@ -189,6 +189,36 @@
     "prefix": {
       "type": "string"
     },
+    "project_reuse": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "use_data_source": {
+          "type": "boolean"
+        },
+        "service_projects": {
```

Comment:
Luca, this should mirror the variable type in the project module. Why do you have `service_projects` here?

Reply:
I agree but and I have no idea. It was't inserted by me.

Reply:
it's in your PR lol...

Reply:
ahhhhh sorry. I thought you were referencing to something different. Saw it now. Apologies.

Reply:
ok done. Let's try again.